ksql-engine/src/test/java/io/confluent/ksql/function/BlacklistTest.java

```diff
@@ -0,0 +1,102 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.function;
+
+import com.google.common.collect.ImmutableList;
+
+import org.apache.kafka.test.TestUtils;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.List;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+
+public class BlacklistTest {
+
+  private File blacklistFile;
+
+  @Before
+  public void before() throws IOException {
+     blacklistFile = TestUtils.tempFile();
+  }
+
+  @Test
+  public void shouldBlackListAllInPackage() throws IOException {
+    writeBlacklist(ImmutableList.of("java.lang"));
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertTrue(blacklist.test("java.lang.Class"));
+    assertFalse(blacklist.test("java.util.List"));
+  }
+
+  @Test
+  public void shouldBlackListClassesMatching() throws IOException {
+    writeBlacklist(ImmutableList.of("java.lang.Process"));
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertTrue(blacklist.test("java.lang.Process"));
+    assertTrue(blacklist.test("java.lang.ProcessBuilder"));
+    assertTrue(blacklist.test("java.lang.ProcessEnvironment"));
+    assertFalse(blacklist.test("java.lang.Class"));
+  }
+
+  @SuppressWarnings("ResultOfMethodCallIgnored")
+  @Test
+  public void shouldBlacklistEverythingIfFailsToLoadFile() {
+    blacklistFile.delete();
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertTrue(blacklist.test("java.lang.Process"));
+    assertTrue(blacklist.test("java.util.List"));
+    assertTrue(blacklist.test("java.lang.ProcessEnvironment"));
+    assertTrue(blacklist.test("java.lang.Class"));
+  }
+
+  @SuppressWarnings("ResultOfMethodCallIgnored")
+  @Test
+  public void shouldNotBlacklistAnythingIfBlacklistFileIsEmpty() {
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertFalse(blacklist.test("java.lang.Process"));
+    assertFalse(blacklist.test("java.util.List"));
+    assertFalse(blacklist.test("java.lang.ProcessEnvironment"));
+    assertFalse(blacklist.test("java.lang.Class"));
+  }
+
+  @Test
+  public void shouldIgnoreBlankLines() throws IOException {
+    writeBlacklist(ImmutableList.<String>builder().add("", "java.util", "").build());
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertFalse(blacklist.test("java.lang.Process"));
+    assertTrue(blacklist.test("java.util.List"));
+  }
+
+  @Test
+  public void shouldIgnoreLinesStartingWithHash() throws IOException {
+    writeBlacklist(ImmutableList.<String>builder().add("#", "java.util", "#").build());
+    final Blacklist blacklist = new Blacklist(this.blacklistFile);
+    assertFalse(blacklist.test("java.lang.String"));
```

Comment:
hamcrrest?

Reply:
i'm happy with `assertFalse` if it testing a boolean

Reply:
Be nice to be consistent and always use hamcrest.